.github/workflows/smoke.yaml

```diff
@@ -139,7 +139,7 @@ jobs:
           go-version: "1.21"
       - env:
           OPENAI_API_KEY: ${{ secrets.SMOKE_OPENAI_API_KEY }}
-          GPTSCRIPT_DEFAULT_MODEL: claude-3-opus-20240229 from github.com/gptscript-ai/claude3-anthropic-provider@tool-beta
+          GPTSCRIPT_DEFAULT_MODEL: claude-3-opus-20240229 from github.com/gptscript-ai/claude3-anthropic-provider
```

Comment:
I think you'll need to do a credential override for this to work now that https://github.com/gptscript-ai/claude3-anthropic-provider/commit/cc53b05ea7649b8ee1a3020f4dc4a5dd11fd7b68 merged, right?

Reply:
Oh good point, I guess I will